core/lib/dal/src/events_dal.rs

```diff
@@ -182,11 +186,65 @@ impl EventsDal<'_, '_> {
         .unwrap();
     }
 
-    pub(crate) async fn l2_to_l1_logs(
+    pub(crate) async fn get_logs_by_hashes(
```

Comment:
Bikeshedding: IMO, `get_logs_by_transaction_hashes` / `get_logs_by_tx_hashes` would be clearer.